### PR TITLE
chore(README): add version table

### PR DESCRIPTION
**Pull request by hezhizhen**
**What this PR does / why we need it**:

chore(README): add version table

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @whalecold 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kube/specified_ip.go [hunk 1]
```diff
@@ -58,8 +58,14 @@ func (c *client) applyIPSpecDecreasing(client *ResourceClient, namespace string,
...
 	tempObj := existence.DeepCopyObject()
-	setRuntimeObjectAnnotationValue(tempObj, AnnoKeySpecifiedIPs, targetAnnoValue)
-	setRuntimeObjectAnnotationValue(tempObj, AnnoKeySpecifiedIPsDecreasing, "") // rm on final update
+	updated := setRuntimeObjectAnnotationValue(tempObj, AnnoKeySpecifiedIPs, targetAnnoValue)
+	if !updated {
+		glog.Infof("fail to set annotation (%s) for runtime object", AnnoKeySpecifiedIPs)
+	}
+	updated = setRuntimeObjectAnnotationValue(tempObj, AnnoKeySpecifiedIPsDecreasing, "") // rm on final update
+	if !updated {
+		glog.Infof("fail to set annotation (%s) for runtime object", AnnoKeySpecifiedIPsDecreasing)
```
Comment (whalecold) [inline pkg/kube/specified_ip.go]: 这里 updated 不是失败的意思，看代码是之前是否已经是最新配置。

Comment (hezhizhen) [reply pkg/kube/specified_ip.go]: 看看新改的？

**Pull request merged**